nodejs_conductor/native/src/config.rs

```diff
@@ -42,9 +42,14 @@ pub fn js_make_config(mut cx: FunctionContext) -> JsResult<JsValue> {
     let logger = if opts.debug_log {
         Default::default()
     } else {
+        // creates a logger which just mutes all
+        let mut rules = LogRules::new();
+        rules
+            .add_rule("^*", true, None)
```

Comment:
the `^` is redundant but makes no difference either way

Reply:
I don't believe that `^*` is a valid Regular Expression, though; the "Kleene star" indicates "match the preceding character 0 or more times", but `^` isn't really a character but an "anchor specifying that the following pattern must begin at the first character of the string". So, a Regular Expression engine could legitimately reject this pattern as invalid, as there is no "character" specified for either the anchor nor the star.  Better to just use `.*` for a universal match.

Reply:
thanks Pj. We can update this